---
annotations:
- id: PW:0000502
  parent: regulatory pathway
  type: Pathway Ontology
  value: complement system pathway
authors:
- Elisa
- Adinasarapu
- Anwesha
- Khanspers
- Egonw
- MaintBot
- Fehrhart
- Mkutmon
- Wpblocked
- AlexanderPico
- Ariutta
- DeSl
- Eweitz
- Larsgw
- Sikeda
citedin:
- link: PMC8357963
  title: 'Looking for pathways related to COVID-19: confirmation of pathogenic mechanisms
    by SARS-CoV-2â€“host interactome (2021)'
- link: PMC8200404
  title: Assessing the Contribution of Relative Macrophage Frequencies to Subcutaneous
    Adipose Tissue (2021)
- link: PMC5085087
  title: Long Term Culture of the A549 Cancer Cell Line Promotes Multilamellar Body
    Formation and Differentiation towards an Alveolar Type II Pneumocyte Phenotype
    (2016)
- link: PMC9734099
  title: Acute and chronic blood serum proteome changes in patients with methanol
    poisoning (2022)
- link: PMC9607846
  title: 'Discovering Common Pathogenic Mechanisms of COVID-19 and Parkinson Disease:
    An Integrated Bioinformatics Analysis (2022)'
- link: 10.1016/j.compbiomed.2021.104243
  title: Construction and analysis of protein-protein interaction network of non-alcoholic
    fatty liver disease
- link: PMC10476796
  title: Demonstration of the impact of COVID-19 on metabolic associated fatty liver
    disease by bioinformatics and system biology approach (2023)
- link: PMC9785216
  title: Comparative RNA-Sequencing Analysis Reveals High Complexity and Heterogeneity
    of Transcriptomic and Immune Profiles in Hepatocellular Carcinoma Tumors of Viral
    (HBV, HCV) and Non-Viral Etiology (2022)
- link: PMC12004383
  title: Proteomics Analysis of Porcine Endometrial Cell-Derived Extracellular Vesicles
    Involved in Embryo Attachment (2025)
- link: PMC11857022
  title: Proteomic Approach to Study the Effect of Pneumocystis jirovecii Colonization
    in Idiopathic Pulmonary Fibrosis (2025)
- link: PMC11821529
  title: Bat genomes illuminate adaptations to viral tolerance and disease resistance
    (2025)
communities: []
description: 'The complement activation takes place through one or more of the well-established
  (alternative, classical or lectin) pathways consisting of plasma and membrane-bound
  proteins. All three pathways converge at the level of complement C3 ([doi:10.6072/H0.MP.A004235.01](https://doi.org/10.6072/H0.MP.A004235.01))
  and are controlled by regulators ([doi:10.1038/ni.1923](https://doi.org/10.1038/ni.1923)).
  Complement C3 belongs to the alpha-2-macroglobulin family of proteins, and consists
  of a alpha-chain and an beta-chain. Cleavage of C3 which can be initiated by one
  or more of the above three distinct pathways, into C3b [Proteolysis@23-667,749-1663]
  and C3a [Proteolysis@672-748] is an important step in the complement activation
  cascade. Classical and lectin pathways, when activated with recognition of pathogens
  (or immune complexes) use C3-convertase [C4b2a] to cleave complement C3 into C3a
  and C3b ([doi:10.1084/jem.125.2.359](https://doi.org/10.1084/jem.125.2.359)). However,
  in alternative pathway a small fraction of the C3 molecules are hydrolyzed to C3(H20)
  exposing new binding sites. This hydrated C3 [C3(H20)] recruits complement factor
  B [fB], which is then cleaved by complement factor D [fD] to result in formation
  of the minor form of C3-convertase [C3(H20)Bb] that cleaves C3 into C3a and C3b
  ([doi:10.1084/jem.154.3.856](https://doi.org/10.1084/jem.154.3.856)). Further, addition
  of C3b to C3 convertase [C3bBb or C4b2a] results in C5 convertase [C3bBb3b or C4b2a3b],
  that cleaves complement C5 to C5a and C5b, is the last enzymatic step of the complement
  activation cascade ([doi:10.1074/jbc.273.27.16828](https://www.ncbi.nlm.nih.gov/pubmed/?term=2387864)).
  During complement activation C5b interacts with complement C6, C7, C8 and C9 in
  a sequential and non-catalyzed manner to result in the formation of Terminal Complement
  Complex (TCC) ([doi:10.1074/jbc.M111.219766](https://doi.org/10.1074/jbc.M111.219766)).
  The entire network is considered as a simple recognition and elimination system
  of host-immune complexes and apoptotic and/or pathogens, and therefore promotes
  host immune homeostasis. The complement system is also involved in cross-talk with
  other processes related to coagulation, lipid metabolism and cancer. However, many
  pathogens counteract complement attack through a range of different mechanisms,
  such acquisition of host complement regulators to the surface of pathogen, or secretion
  of complement inactivation factors. In order to have a holistic view of the entire
  complement network, Dr. John D.Lambris group (University of Pennsylvania) developed
  the Complement Map Database (CMAP) which is a unique repository focused on documented
  molecular interactions described within the complement cascade and between complement
  and other biological systems. Information contained in [CMAP](http://www.complement.us/cmap/index.php)
  (see doi:[10.1093/bioinformatics/btt269](https://doi.org/10.1093/bioinformatics/btt269))
  is entirely based on published experimental data and is fully revised by experts
  in the field. Further, the [Signaling Gateway Molecule Pages](https://escholarship.org/uc/molecule_pages)
  (doi:[10.1093/bioinformatics/btr190](https://doi.org/10.1093/bioinformatics/btr190]))
  has published a curated data on each protein involved in human complement activation
  pathways (refs. [Dinasarapu et al](https://doi.org/10.6072/H0.MP.A004235.01), [Chandrasekhar
  et al](https://doi.org/10.6072/H0.MP.A004228.01), [Dinasarapu et al](https://doi.org/10.6072/H0.MP.A004276.01),
  [Dinasarapu et al](https://doi.org/10.6072/H0.MP.A004256.01), [Chandrasekhar et
  al](https://doi.org/10.6072/H0.MP.A004240.01), [Chandrasekhar et al](https://doi.org/10.6072/H0.MP.A008392.01),
  [Chandrasekhar et al](https://doi.org/10.6072/H0.MP.A008391.01), [Chandrasekhar
  et al](https://doi.org/10.6072/H0.MP.A004274.01), [Chandrasekhar et al](https://doi.org/10.6072/H0.MP.A004275.01),
  [Chandrasekhar et al](https://doi.org/10.6072/H0.MP.A004266.01), [Chandrasekhar
  et al](https://doi.org/10.6072/H0.MP.A004267.01), [Dinasarapu et al](https://doi.org/10.6072/H0.MP.A004263.01),
  [Dinasarapu et al](https://doi.org/10.6072/H0.MP.A004234.01), [Min et al](https://doi.org/10.6072/H0.MP.A004258.01)). '
last-edited: 2025-03-19
ndex: 8af76961-8b65-11eb-9e72-0ac135e8bacf
organisms:
- Homo sapiens
redirect_from:
- /index.php/Pathway:WP2806
- /instance/WP2806
- /instance/WP2806_r138047
revision: r138047
schema-jsonld:
- '@context': https://schema.org/
  '@id': https://wikipathways.github.io/pathways/WP2806.html
  '@type': Dataset
  creator:
    '@type': Organization
    name: WikiPathways
  description: 'The complement activation takes place through one or more of the well-established
    (alternative, classical or lectin) pathways consisting of plasma and membrane-bound
    proteins. All three pathways converge at the level of complement C3 ([doi:10.6072/H0.MP.A004235.01](https://doi.org/10.6072/H0.MP.A004235.01))
    and are controlled by regulators ([doi:10.1038/ni.1923](https://doi.org/10.1038/ni.1923)).
    Complement C3 belongs to the alpha-2-macroglobulin family of proteins, and consists
    of a alpha-chain and an beta-chain. Cleavage of C3 which can be initiated by one
    or more of the above three distinct pathways, into C3b [Proteolysis@23-667,749-1663]
    and C3a [Proteolysis@672-748] is an important step in the complement activation
    cascade. Classical and lectin pathways, when activated with recognition of pathogens
    (or immune complexes) use C3-convertase [C4b2a] to cleave complement C3 into C3a
    and C3b ([doi:10.1084/jem.125.2.359](https://doi.org/10.1084/jem.125.2.359)).
    However, in alternative pathway a small fraction of the C3 molecules are hydrolyzed
    to C3(H20) exposing new binding sites. This hydrated C3 [C3(H20)] recruits complement
    factor B [fB], which is then cleaved by complement factor D [fD] to result in
    formation of the minor form of C3-convertase [C3(H20)Bb] that cleaves C3 into
    C3a and C3b ([doi:10.1084/jem.154.3.856](https://doi.org/10.1084/jem.154.3.856)).
    Further, addition of C3b to C3 convertase [C3bBb or C4b2a] results in C5 convertase
    [C3bBb3b or C4b2a3b], that cleaves complement C5 to C5a and C5b, is the last enzymatic
    step of the complement activation cascade ([doi:10.1074/jbc.273.27.16828](https://www.ncbi.nlm.nih.gov/pubmed/?term=2387864)).
    During complement activation C5b interacts with complement C6, C7, C8 and C9 in
    a sequential and non-catalyzed manner to result in the formation of Terminal Complement
    Complex (TCC) ([doi:10.1074/jbc.M111.219766](https://doi.org/10.1074/jbc.M111.219766)).
    The entire network is considered as a simple recognition and elimination system
    of host-immune complexes and apoptotic and/or pathogens, and therefore promotes
    host immune homeostasis. The complement system is also involved in cross-talk
    with other processes related to coagulation, lipid metabolism and cancer. However,
    many pathogens counteract complement attack through a range of different mechanisms,
    such acquisition of host complement regulators to the surface of pathogen, or
    secretion of complement inactivation factors. In order to have a holistic view
    of the entire complement network, Dr. John D.Lambris group (University of Pennsylvania)
    developed the Complement Map Database (CMAP) which is a unique repository focused
    on documented molecular interactions described within the complement cascade and
    between complement and other biological systems. Information contained in [CMAP](http://www.complement.us/cmap/index.php)
    (see doi:[10.1093/bioinformatics/btt269](https://doi.org/10.1093/bioinformatics/btt269))
    is entirely based on published experimental data and is fully revised by experts
    in the field. Further, the [Signaling Gateway Molecule Pages](https://escholarship.org/uc/molecule_pages)
    (doi:[10.1093/bioinformatics/btr190](https://doi.org/10.1093/bioinformatics/btr190]))
    has published a curated data on each protein involved in human complement activation
    pathways (refs. [Dinasarapu et al](https://doi.org/10.6072/H0.MP.A004235.01),
    [Chandrasekhar et al](https://doi.org/10.6072/H0.MP.A004228.01), [Dinasarapu et
    al](https://doi.org/10.6072/H0.MP.A004276.01), [Dinasarapu et al](https://doi.org/10.6072/H0.MP.A004256.01),
    [Chandrasekhar et al](https://doi.org/10.6072/H0.MP.A004240.01), [Chandrasekhar
    et al](https://doi.org/10.6072/H0.MP.A008392.01), [Chandrasekhar et al](https://doi.org/10.6072/H0.MP.A008391.01),
    [Chandrasekhar et al](https://doi.org/10.6072/H0.MP.A004274.01), [Chandrasekhar
    et al](https://doi.org/10.6072/H0.MP.A004275.01), [Chandrasekhar et al](https://doi.org/10.6072/H0.MP.A004266.01),
    [Chandrasekhar et al](https://doi.org/10.6072/H0.MP.A004267.01), [Dinasarapu et
    al](https://doi.org/10.6072/H0.MP.A004263.01), [Dinasarapu et al](https://doi.org/10.6072/H0.MP.A004234.01),
    [Min et al](https://doi.org/10.6072/H0.MP.A004258.01)). '
  keywords:
  - A2M
  - ADIPOQ
  - ADM
  - ALB
  - APOA1
  - ARRB2
  - C1INH
  - C1qRp
  - C1r
  - C1s
  - C2
  - C3
  - C3(H2O)
  - C3a
  - C3a-desArg
  - C3aR1
  - C3b
  - C3c
  - C3d
  - C3f
  - C4-A
  - C4BP
  - C4a
  - C4a-desArg
  - C4b
  - C5
  - C5L2
  - C5a
  - C5a-desArg
  - C5aR1
  - C5b
  - C6
  - C7
  - C8
  - C9
  - CD16a
  - CD19
  - CD23
  - CD40
  - CD59
  - CFB
  - CFD
  - CFH
  - CFHR1
  - CFHR4
  - CFI
  - CFP
  - CHIPS
  - CK
  - CLEC4M
  - 'CPB2 '
  - CPN
  - CR1
  - CR2
  - CRIg
  - CRP
  - CbpA
  - ClfA
  - ClfB
  - DAF
  - DCN
  - E-LDL
  - ELANE
  - Efb
  - Ehp
  - FCN1
  - FCN2
  - FGA
  - FGB
  - FGG
  - FKBP13
  - FN
  - FPR1
  - FX
  - FXIII
  - FXIIa
  - FXIa
  - FnBPA
  - FnBPB
  - GNA15
  - GNA1870
  - GNAI2
  - GNAI3
  - HIV-gp41
  - HS
  - HSV-gC
  - Heparin
  - IBSP
  - ICAM1
  - ICAM2
  - ITGA2
  - ITGA2B
  - ITGB3
  - KLKB1
  - LAMA5
  - LAMB1
  - LAMC1
  - LRP2
  - LTA
  - LfhA
  - MASP1
  - MAp19
  - MAp44
  - MBL2
  - MCP
  - OMCI
  - OPN
  - OmpK36
  - PKA
  - PKC
  - PLAUR
  - PLG
  - PTX3
  - PrP
  - Protein M
  - Protein-A
  - Protein-S
  - RPS19
  - RgpA
  - SAP
  - SCIN
  - SCP
  - SELE
  - SELL
  - SELP
  - SELPLG
  - SFTPA1
  - SFTPA2
  - SIC
  - SSL10
  - SSL11
  - SSL5
  - SSL7
  - Sbi
  - Sfb
  - Skp
  - StcE
  - THBS
  - TLR2
  - TXN
  - 'VCP '
  - VTN
  - WAS
  - beta-Glucan
  - cC1qR
  - fMLP
  - gC1qR
  - gp350
  - iC3b
  license: CC0
  name: Complement system
seo: CreativeWork
title: Complement system
wpid: WP2806
---