---
annotations:
- id: PW:0000032
  parent: classic metabolic pathway
  type: Pathway Ontology
  value: pyrimidine metabolic pathway
authors:
- Annahoekstra1
- DeSl
- Khanspers
- MaintBot
- Finterly
- Eweitz
- Egonw
citedin:
- link: PMC8742820
  title: DTYMK is essential for genome integrity and neuronal survival (2021)
- link: PMC8751594
  title: DNA methylation of ARHGAP30 is negatively associated with ARHGAP30 expression
    in lung adenocarcinoma, which reduces tumor immunity and is detrimental to patient
    survival (2021)
- link: 10.1038/s41467-024-47085-y
  title: A patient-based iPSC-derived hepatocyte model of alcohol-associated cirrhosis
    reveals bioenergetic insights into disease pathogenesis (2024)
- link: 10.1159/000535120
  title: Human Monocytes Exposed to SARS-CoV-2 Display Features of Innate Immune Memory
    Producing High Levels of CXCL10 upon Restimulation (2023)
- link: 10.1186/s13023-023-02683-9
  title: Extending inherited metabolic disorder diagnostics with biomarker interaction
    visualizations (2023)
- link: PMC12301734
  title: Investigating Transcriptional Age Acceleration in Inflammatory Skin Diseases
communities:
- MetaKids
description: 'Pyrimidines are nucleic acids and the products of pyrimidine degradation
  are water-soluble. The pyrimidine ring is synthesized before it is conjugated to
  PRPP. The first reaction is the conjugation of carbamoyl phosphate and aspartate
  to make N-carbamoylaspartate. The carbamoyl phosphate synthetase used in pyrimidine
  biosynthesis is located in the cytoplasm. The enzyme that carries out the reaction
  is aspartate transcarbamoylase, an enzyme that is closely regulated. The second
  reaction is ring closure to form dihydroorotic acid by the enzyme dihydroorotase.
  This circular product contains a 6-membered ring with nitrogen and carbons located
  in the same positions as in the mature pyrimidine ring. The third reaction is the
  oxidation of the ring to form a carbon- carbon bond. The reducing equivalents are
  transferred to a flavin cofactor of the enzyme dihydroorotate dehydrogenase. The
  product is orotic acid. Fourth, the orotate ring is transferred to phosphoribosyl
  pyrophosphate (PRPP) to form a 5-ribose-phosphate, orotidylic acid. Finally orotidylate
  is decarboxylated to yield UMP, which of course contains one of the bases of RNA.
  Cellular kinases convert UMP to UTP. Transfer of an amido nitrogen from glutamine
  by CTP synthetase converts UTP to CTP; this reaction uses an ATP high-energy phosphate.
  Pyrimidine synthesis is controlled at the first committed step. ATP stimulates the
  aspartate transcarbamoylase reaction, while CTP inhibits it. CTP is a feedback inhibitor
  of the pathway, and ATP is a feed-forward activator. This regulation ensures that
  a balanced supply of purines and pyrimidines exists for RNA and synthesis. Eukaryotic
  organisms contain a multifunctional enzyme with carbamoylphosphate synthetase, aspartate
  transcarbamoylase, and dihydroorotase activities. Two mechanisms control this enzyme.
  First, control at the level of enzyme synthesis exists; the transcription of the
  gene for the enzyme is reduced if an excess of pyrimidines is present. Secondly,
  control exists at the level of feedback inhibition by pyrimidine nucleotides. This
  enzyme is also an example of the phenomenon of metabolic channeling: aspartate,
  ammonia, and carbon dioxide enter the enzyme and come out as orotic acid  Description
  text is based on [Cliff''s Notes](https://www.cliffsnotes.com/study-guides/biology/biochemistry-ii/purines-and-pyrimidines/pyrimidine-metabolism).  Pathway
  is based on [KEGG](https://www.genome.jp/dbget-bin/www_bget?pathway+map00240).  Proteins
  on this pathway have targeted assays available via the [CPTAC Assay Portal](https://assays.cancer.gov/available_assays?wp_id=WP4022).'
last-edited: 2025-03-07
ndex: 296f0325-8b69-11eb-9e72-0ac135e8bacf
organisms:
- Homo sapiens
redirect_from:
- /index.php/Pathway:WP4022
- /instance/WP4022
- /instance/WP4022_r137677
revision: r137677
schema-jsonld:
- '@context': https://schema.org/
  '@id': https://wikipathways.github.io/pathways/WP4022.html
  '@type': Dataset
  creator:
    '@type': Organization
    name: WikiPathways
  description: 'Pyrimidines are nucleic acids and the products of pyrimidine degradation
    are water-soluble. The pyrimidine ring is synthesized before it is conjugated
    to PRPP. The first reaction is the conjugation of carbamoyl phosphate and aspartate
    to make N-carbamoylaspartate. The carbamoyl phosphate synthetase used in pyrimidine
    biosynthesis is located in the cytoplasm. The enzyme that carries out the reaction
    is aspartate transcarbamoylase, an enzyme that is closely regulated. The second
    reaction is ring closure to form dihydroorotic acid by the enzyme dihydroorotase.
    This circular product contains a 6-membered ring with nitrogen and carbons located
    in the same positions as in the mature pyrimidine ring. The third reaction is
    the oxidation of the ring to form a carbon- carbon bond. The reducing equivalents
    are transferred to a flavin cofactor of the enzyme dihydroorotate dehydrogenase.
    The product is orotic acid. Fourth, the orotate ring is transferred to phosphoribosyl
    pyrophosphate (PRPP) to form a 5-ribose-phosphate, orotidylic acid. Finally orotidylate
    is decarboxylated to yield UMP, which of course contains one of the bases of RNA.
    Cellular kinases convert UMP to UTP. Transfer of an amido nitrogen from glutamine
    by CTP synthetase converts UTP to CTP; this reaction uses an ATP high-energy phosphate.
    Pyrimidine synthesis is controlled at the first committed step. ATP stimulates
    the aspartate transcarbamoylase reaction, while CTP inhibits it. CTP is a feedback
    inhibitor of the pathway, and ATP is a feed-forward activator. This regulation
    ensures that a balanced supply of purines and pyrimidines exists for RNA and synthesis.
    Eukaryotic organisms contain a multifunctional enzyme with carbamoylphosphate
    synthetase, aspartate transcarbamoylase, and dihydroorotase activities. Two mechanisms
    control this enzyme. First, control at the level of enzyme synthesis exists; the
    transcription of the gene for the enzyme is reduced if an excess of pyrimidines
    is present. Secondly, control exists at the level of feedback inhibition by pyrimidine
    nucleotides. This enzyme is also an example of the phenomenon of metabolic channeling:
    aspartate, ammonia, and carbon dioxide enter the enzyme and come out as orotic
    acid  Description text is based on [Cliff''s Notes](https://www.cliffsnotes.com/study-guides/biology/biochemistry-ii/purines-and-pyrimidines/pyrimidine-metabolism).  Pathway
    is based on [KEGG](https://www.genome.jp/dbget-bin/www_bget?pathway+map00240).  Proteins
    on this pathway have targeted assays available via the [CPTAC Assay Portal](https://assays.cancer.gov/available_assays?wp_id=WP4022).'
  keywords:
  - (R)-3-aminoisobutyrate
  - (R)-3-ureidoisobutyrate
  - (R)-dihydrothymine
  - 2-deoxy-D-ribose-1P
  - 3-Ureidopropionate
  - 5,6-Dihydrouracil
  - AK9
  - CAD
  - CANT1
  - CDA
  - CDP
  - CMP
  - CMPK1
  - CMPK2
  - CTP
  - CTPS1
  - CTPS2
  - Carbamoylphosphate
  - Cytidine
  - DCK
  - DCTD
  - DCTPP1
  - DHODH
  - DNA
  - DPYD
  - DPYS
  - DTYMK
  - DUT
  - Deoxycytidine
  - Deoxyuridine
  - Dihydroorotate
  - Diphosphate
  - ENPP1
  - ENPP3
  - ENTPD1
  - ENTPD3
  - ENTPD4
  - ENTPD5
  - ENTPD6
  - ENTPD8
  - L-Glutamine
  - N-Carbamoylaspartate
  - NME
  - NME1
  - NME1-NME2
  - NME2
  - NME3
  - NME4
  - NME6
  - NME7
  - NT5C
  - NT5E
  - NT5M
  - NUDT2
  - Orotate
  - Orotidine 5'-phosphate
  - PNP
  - PNPT1
  - POLA1
  - POLA2
  - POLD1
  - POLD2
  - POLD3
  - POLD4
  - POLE1
  - POLE2
  - POLE3
  - POLE4
  - POLR1A
  - POLR1B
  - POLR1C
  - POLR1D
  - POLR1E
  - POLR2A
  - POLR2B
  - POLR2C
  - POLR2D
  - POLR2E
  - POLR2G
  - POLR2H
  - POLR2I
  - POLR2J1
  - POLR2J2
  - POLR2J3
  - POLR2K
  - POLR2L
  - POLR3A
  - POLR3B
  - POLR3C
  - POLR3D
  - POLR3E
  - POLR3F
  - POLR3G
  - POLR3GL
  - POLR3H
  - POLR3K
  - PRIM1
  - PRIM2
  - PRPP
  - RNA
  - RRM1
  - RRM2
  - RRM2B
  - TK1
  - TK2
  - TWISTNB
  - TYMP
  - TYMS
  - Thymidine
  - Thymine
  - UCK1
  - UCK2
  - UCKL1
  - UDP
  - UMP
  - UMPS
  - UPB1
  - UPP1
  - UPP2
  - UPRT
  - UTP
  - UppppU
  - Uracil
  - Uridine
  - ZNRD1
  - beta-Alanine
  - dCDP
  - dCMP
  - dCTP
  - dTDP
  - dTMP
  - dTTP
  - dUDP
  - dUMP
  - dUTP
  license: CC0
  name: Pyrimidine metabolism
seo: CreativeWork
title: Pyrimidine metabolism
wpid: WP4022
---