---
annotations:
- id: DOID:934
  parent: disease by infectious agent
  type: Disease Ontology
  value: viral infectious disease
- id: DOID:0080600
  parent: disease by infectious agent
  type: Disease Ontology
  value: COVID-19
- id: PW:0000003
  parent: signaling pathway
  type: Pathway Ontology
  value: signaling pathway
- id: PW:0000895
  parent: signaling pathway
  type: Pathway Ontology
  value: type I interferon signaling pathway
- id: DOID:2945
  parent: disease by infectious agent
  type: Disease Ontology
  value: severe acute respiratory syndrome
authors:
- AlexanderPico
- Egonw
- Fehrhart
- NhungP
- DeSl
- AnnaNi
- Eweitz
- Finterly
- Mkutmon
citedin:
- link: PMC9519890
  title: 'Tissue-specific pathway activities: A retrospective analysis in COVID-19
    patients (2022)'
- link: PMC9537444
  title: Bioinformatics and systems-biology analysis to determine the effects of Coronavirus
    disease 2019 on patients with allergic asthma (2022)
- link: 10.3389/fcimb.2023.1280223
  title: Discovering common pathogenetic processes between COVID-19 and tuberculosis
    by bioinformatics and system biology approach (2023)
- link: 10.1155/2022/3515001
  title: Combination of Enrichment Using Gene Ontology and Transcriptomic Analysis
    Revealed Contribution of Interferon Signaling to Severity of COVID-19 (2022)
- link: 10.1038/s41586-024-07873-4
  title: Fibrin drives thromboinflammation and neuropathology in COVID-19 (2024)
- link: 10.3390/ijms25115731
  title: Longitudinal Neuropathological Consequences of Extracranial Radiation Therapy
    in Mice (2024)
- link: 10.3390/ijms25115731
  title: Longitudinal Neuropathological Consequences of Extracranial Radiation Therapy
    in Mice (2024)
- link: 10.1016/j.stemcr.2023.05.007
  title: Parallel use of human stem cell lung and heart models provide insights for
    SARS-CoV-2 treatment (2023)
- link: PMC11951896
  title: 'Dynamic Gene Attention Focus (DyGAF): Enhancing Biomarker Identification
    Through Dual-Model Attention Networks'
- link: PMC12323659
  title: Decoding the transcriptome from bulk RNA of infection-na√Øve versus imprinted
    patients with SARS-CoV-2 Omicron B.1.1.529 (2025)
communities:
- COVID19
description: The induction of Type I interferons and signaling is the first response
  leading to the innate immune reactions during SARS-COV-2 infection. The virus can
  enter host cells through two mechanisms. If it enters the cell via diffusion mediated
  by TMPRSS2, the virus ssRNA will be detected by RIG-I and MDA5 in the cytosol. If
  the virus enters the cell via endocytosis, the spike proteins will be processed
  by CTSL in the lysosome leading to the detection of ssRNA by TLR3,7 and 9 (PMID
  33506952). The extracellular virus can also be detected by TLR2,4 and 6 (PMID 33506952).
  The higher production of TLR4 in men and the presence of TLR7 on the X chromosome
  may contribute to the different responses between women and men during SARS-CoV
  2 infection (PMID 33506952).   TLR7 MYD88-dependent signaling is inhibited at multiple
  steps by the SARS-CoV Papain-Like Protease (PLpro) domain of nsp3 (red oval). The
  signaling pathway is critical to induction of type I interferons (INF-I) via IRF3,
  AP-1 and NFkB transcription factors. INF-I triggers the JAK/STAT pathway leading
  to the induction of interferon-stimulated genes (ISGs), such as OAS and PKR, which
  go one to conduct the innate immune response. TREML4 has been shown to be necessary
  for MYD88 recruitment by TLR7 and STAT1 participation. The inhibition of SARS-CoV-2
  PLpro by GRL0617 is proposed based on Ratia, et al. 2008 and 100% sequence identity
  between SARS-CoV and SARS-CoV-2 across all 13 residues of PLpro involved in binding
  GRL0617 (82.9% identity across 316 amino acids) as determined by the alignment of
  RefSeq YP_009725299.1 and PDB 3E9S (https://alexanderpico.github.io/SARS-CoV-2_Alignments/#Nsp3_PLpro_domain).
  The antimicrobial agent, azithromycin, is in clincal trials as COVID-19 therapy
  in combination with hydroxychloroquine (Gautret 2020) has been shown to modulate
  inflammation by inhibiting the activation of many of these same transcription factors.
last-edited: 2024-10-30
ndex: 9f30ac89-8b6f-11eb-9e72-0ac135e8bacf
organisms:
- Homo sapiens
redirect_from:
- /index.php/Pathway:WP4868
- /instance/WP4868
- /instance/WP4868_r135713
revision: r135713
schema-jsonld:
- '@context': https://schema.org/
  '@id': https://wikipathways.github.io/pathways/WP4868.html
  '@type': Dataset
  creator:
    '@type': Organization
    name: WikiPathways
  description: The induction of Type I interferons and signaling is the first response
    leading to the innate immune reactions during SARS-COV-2 infection. The virus
    can enter host cells through two mechanisms. If it enters the cell via diffusion
    mediated by TMPRSS2, the virus ssRNA will be detected by RIG-I and MDA5 in the
    cytosol. If the virus enters the cell via endocytosis, the spike proteins will
    be processed by CTSL in the lysosome leading to the detection of ssRNA by TLR3,7
    and 9 (PMID 33506952). The extracellular virus can also be detected by TLR2,4
    and 6 (PMID 33506952). The higher production of TLR4 in men and the presence of
    TLR7 on the X chromosome may contribute to the different responses between women
    and men during SARS-CoV 2 infection (PMID 33506952).   TLR7 MYD88-dependent signaling
    is inhibited at multiple steps by the SARS-CoV Papain-Like Protease (PLpro) domain
    of nsp3 (red oval). The signaling pathway is critical to induction of type I interferons
    (INF-I) via IRF3, AP-1 and NFkB transcription factors. INF-I triggers the JAK/STAT
    pathway leading to the induction of interferon-stimulated genes (ISGs), such as
    OAS and PKR, which go one to conduct the innate immune response. TREML4 has been
    shown to be necessary for MYD88 recruitment by TLR7 and STAT1 participation. The
    inhibition of SARS-CoV-2 PLpro by GRL0617 is proposed based on Ratia, et al. 2008
    and 100% sequence identity between SARS-CoV and SARS-CoV-2 across all 13 residues
    of PLpro involved in binding GRL0617 (82.9% identity across 316 amino acids) as
    determined by the alignment of RefSeq YP_009725299.1 and PDB 3E9S (https://alexanderpico.github.io/SARS-CoV-2_Alignments/#Nsp3_PLpro_domain).
    The antimicrobial agent, azithromycin, is in clincal trials as COVID-19 therapy
    in combination with hydroxychloroquine (Gautret 2020) has been shown to modulate
    inflammation by inhibiting the activation of many of these same transcription
    factors.
  keywords:
  - ACE2
  - Azithromycin
  - GRL0617
  - IFNAR1
  - IFNAR2
  - IKBKE
  - INF-I alpha/ beta
  - IRAK4
  - IRF3
  - IRF7
  - IRF9
  - JAK1
  - MAVS
  - 'MDA5 '
  - MYD88
  - OAS1
  - OAS2
  - OAS3
  - PKR
  - PLpro (nsp3)
  - RIG-I (DDX58)
  - STAT1
  - STAT2
  - TBK1
  - TLR2
  - TLR3
  - TLR4
  - TLR6
  - TLR7
  - TLR9
  - TMPRSS2
  - TRAF3
  - TRAF6
  - 'TREML4 '
  - TYK2
  - nsp1
  - nsp10
  - nsp13
  - nsp14
  - nsp15
  - nsp16
  - orf3a
  - orf6
  license: CC0
  name: Type I interferon induction and signaling during SARS-CoV-2 infection
seo: CreativeWork
title: Type I interferon induction and signaling during SARS-CoV-2 infection
wpid: WP4868
---