---
annotations:
- id: DOID:0080600
  parent: disease by infectious agent
  type: Disease Ontology
  value: COVID-19
- id: DOID:0050117
  parent: disease by infectious agent
  type: Disease Ontology
  value: disease by infectious agent
- id: PW:0000013
  parent: disease pathway
  type: Pathway Ontology
  value: disease pathway
authors:
- Egonw
- L Dupuis
- Evelo
- AlexanderPico
- MaintBot
- Fehrhart
- Mkutmon
- Eweitz
- NhungP
citedin:
- link: PMC9326261
  title: Multi-Data Integration Towards a Global Understanding of the Neurological
    Impact of Human Brain Severe Acute Respiratory Syndrome Coronavirus 2 Infection
    (2022)
- link: PMC7820539
  title: 'A protocol for adding knowledge to Wikidata: aligning resources on human
    coronaviruses (2021)'
- link: PMC9627153
  title: 'Characterization of the SARS-CoV-2 co-receptor NRP1 expression profiles
    in healthy people and cancer patients: Implication for susceptibility to COVID-19
    disease and potential therapeutic strategy (2022)'
- link: PMC9087427
  title: 'Social Determinants of Health Factors for Gene–Environment COVID‐19 Research:
    Challenges and Opportunities (2022)'
- link: PMC9519890
  title: 'Tissue-specific pathway activities: A retrospective analysis in COVID-19
    patients (2022)'
- link: 10.3390/ijms241713377
  title: The Influence of KE and EW Dipeptides in the Composition of the Thymalin
    Drug on Gene Expression and Protein Synthesis Involved in the Pathogenesis of
    COVID-19 (2023)
- link: 10.3390/v16010100
  title: Investigating the Potential Shared Molecular Mechanisms between COVID-19
    and Alzheimer’s Disease via Transcriptomic Analysis (2023)
communities:
- COVID19
description: 'Collaborative project for curation biological processes involved in
  the COVID-19 disease after SARS-Cov-2 infection. It focuses on experimental evidence
  and plays with improved annotation of complexes and with the Evidence and Conclusion
  Ontology. The complexes link to EBI''s Complex Portal, resulting from a collaboration
  with that database at the recent online ELIXIR biohackathon. Editing this pathway
  is (at this moment) coordinated via the wikipathways.slack.com #sarscov2 channel.
  Additionally, please feel free to add suggestions to the discussion page (see the
  tab at the top of this page). The large viral Spike protein (S or surface glycoprotein)
  forms trimers. It interacts with the host''s ACE2 receptor to establish binding
  (Hoffmann et al 2020). There are suggestions for more than one cell entry mechanism,
  with the evidence for ACE2/TMPRSS2 entry being most clear now. Lack of expression
  of TMPRSS2 may explain age differences in COVID19 severity. In this mechanism, to
  enter the virus needs to be primed by the host protease TMPRSS2 that splits the
  Spike protein into 2 peptides S1 and S2. S1 contains the ACE2 receptor binding site,
  S2 binds to the host cell membrane which leads to membrane fusion, the start of
  the uptake process. The ACE2 receptor interaction was also suggested as the start
  of specific lung-damaging effects. Other human genes that may be involved in alternative
  cell uptake mechanisms include CTSL and SLC6A19.'
last-edited: 2025-08-06
ndex: cee0461a-8b6e-11eb-9e72-0ac135e8bacf
organisms:
- Homo sapiens
redirect_from:
- /index.php/Pathway:WP4846
- /instance/WP4846
- /instance/WP4846_r140186
revision: r140186
schema-jsonld:
- '@context': https://schema.org/
  '@id': https://wikipathways.github.io/pathways/WP4846.html
  '@type': Dataset
  creator:
    '@type': Organization
    name: WikiPathways
  description: 'Collaborative project for curation biological processes involved in
    the COVID-19 disease after SARS-Cov-2 infection. It focuses on experimental evidence
    and plays with improved annotation of complexes and with the Evidence and Conclusion
    Ontology. The complexes link to EBI''s Complex Portal, resulting from a collaboration
    with that database at the recent online ELIXIR biohackathon. Editing this pathway
    is (at this moment) coordinated via the wikipathways.slack.com #sarscov2 channel.
    Additionally, please feel free to add suggestions to the discussion page (see
    the tab at the top of this page). The large viral Spike protein (S or surface
    glycoprotein) forms trimers. It interacts with the host''s ACE2 receptor to establish
    binding (Hoffmann et al 2020). There are suggestions for more than one cell entry
    mechanism, with the evidence for ACE2/TMPRSS2 entry being most clear now. Lack
    of expression of TMPRSS2 may explain age differences in COVID19 severity. In this
    mechanism, to enter the virus needs to be primed by the host protease TMPRSS2
    that splits the Spike protein into 2 peptides S1 and S2. S1 contains the ACE2
    receptor binding site, S2 binds to the host cell membrane which leads to membrane
    fusion, the start of the uptake process. The ACE2 receptor interaction was also
    suggested as the start of specific lung-damaging effects. Other human genes that
    may be involved in alternative cell uptake mechanisms include CTSL and SLC6A19.'
  keywords:
  - 25HC
  - 3CL-PRO
  - ACAT
  - ACE2
  - CTSL
  - E
  - ExoN
  - FURIN
  - HDL
  - M
  - N
  - NRP1
  - ORF10
  - ORF3a
  - ORF6
  - ORF7a
  - ORF7b
  - ORF8
  - PL2-PRO
  - S
  - 'S2 '
  - SARS-CoV-2proteins
  - SCARB1
  - SLC6A19
  - TLR7
  - TMPRSS2
  - TMPRSS4
  - cholesterol
  - envelopeprotein
  - heparan sulfate
  - membraneglycoprotein
  - nsp1
  - nsp10
  - nsp12
  - nsp13
  - nsp15
  - nsp16
  - nsp2
  - nsp4
  - nsp5
  - nsp6
  - nsp7
  - nsp8
  - nsp9
  - nucleocapsidphosphoprotein
  - orf1
  - orf10
  - orf14
  - orf1a
  - orf1ab
  - orf3a
  - orf6
  - orf7a
  - orf7b
  - orf8
  - sphingosine
  - surfaceglycoprotein
  license: CC0
  name: SARS-CoV-2 and COVID-19 pathway
seo: CreativeWork
title: SARS-CoV-2 and COVID-19 pathway
wpid: WP4846
---