---
annotations:
- id: PW:0000204
  parent: signaling pathway
  type: Pathway Ontology
  value: Notch signaling pathway
authors:
- A.Pandey
- MaintBot
- Khanspers
- NetPath
- DeSl
- L Dupuis
- Egonw
- Eweitz
citedin:
- link: PMC9317794
  title: A Novel Based-Network Strategy to Identify Phytochemicals from Radix Salviae
    Miltiorrhizae (Danshen) for Treating Alzheimer’s Disease (2022)
- link: 10.1016/j.compbiomed.2014.12.003
  title: Integrative epigenetic profiling analysis identifies DNA methylation changes
    associated with chronic alcohol consumption (2015)
communities:
- CPTAC
- ONTOX
description: 'The Notch pathway is an evolutionally conserved signaling pathway which
  plays an important role in diverse developmental and physiological processes. These
  include cell-fate determination, tissue patterning and morphogenesis, cell differentiation,
  proliferation and cell death. The Notch pathway is named after the Drosophila mutants
  that showed irregular notches of missing tissue at the insect wing blade tips. The
  Notch gene was cloned in 1985. Proteins of the Notch families are single-pass transmembrane
  proteins that function both as cell surface receptors and nuclear transcriptional
  regulators. Four Notch receptors (Notch 1-4) have been identified in mammals. Mature
  Notch receptors are non-covalent heterodimers consisting of an extracellular subunit
  (NEC) and a transmembrane subunit (NTM). NEC possess multiple EGF-like repeats and
  three specialized Lin-Notch repeats (LNR) that forms a tight hydrophobic interaction
  with extracellular stump of NTM. This region masks an A disintegrin and metalloprotease
  (ADAM) cleavage site. The region where these two subunits interact is called the
  heterodimerization domain (HD). Notch ligands are also transmembrane proteins with
  multiple EGF-like repeats, a short cytoplasmic tail and a specialized delta-serrate-lag2
  (DSL) domain at the N-terminus. There are five canonical Notch ligands i.e. Jagged
  (JAG1 and JAG2), Delta-like (DLL1, DLL3, DLL4) in mammals. Notch signaling activation
  occurs upon ligand-receptor binding, which are expressed on two adjacent cells.
  Ligand binding causes dissociation of NEC from NTM, unmasking the ADAM cleavage
  site. The NEC fragment is trans- endocytosed into the ligand expressing cells. The
  full-length receptor minus the NEC fragment is cleaved at the membrane by ADAM17
  generating an intermediate, Notch extracellular truncation (NEXT). This is further
  cleaved by Î³-secretase that generates an active Notch intracellular fragment (NIC)
  or Notch intracellular domain (NICD). The Î³-secretase complex is composed of PSEN1,
  PSEN2, PSENEN, NCSTN and APH1 (A or B). Following these two cleavage steps, the
  NICD is released into the cytoplasm and translocates into the nucleus to regulate
  transcription of Notch target genes. Upon translocation into the nucleus, NICD binds
  to RBPJ which is a constitutive repressor of Notch signaling. RBPJ represses Notch
  target gene expression by recruiting a co-repressor complex, which includes NCOR1,
  NCOR2, SNW1, CIR, HDAC1, HDAC2, SPEN and FHL1 and SAP30. NICD binding to RBPJ replaces
  the co-repressor complex with a co-activator complex which includes MAML1-3, EP300
  and SNW1. Primary Notch target genes include two families of transcriptional factors
  Hes, including HES1 and HES5 as well as Hey including HEY1 and HEY2. Other Notch
  target genes include CCND1, CDKN1A, GATA3 and PTCRA. CNTN1 acts as a functional
  ligand of Notch. This trans-extracellular interaction causes Î³-secretase-dependent
  nuclear translocation of the NICD. This signaling is involved in oligodendrocyte
  precursor cell differentiation and upregulation of myelin-related protein MAG. In
  addition to the canonical Notch pathway, there is increasing evidence showing RBPJ
  independent non-canonical pathways. been fully characterized. Physical interaction
  of NOTCH-1IC with LCK- PI3K may mediate non-nuclear cross-talk with AKT, leading
  to survival signaling. Notch stimulation through AKT pathway leads to down regulation
  of MYC expression. Activation of SRC/STAT3 pathway by Notch signaling is dependent
  on the expression of Notch effector HES1 transcription factor. The induction of
  HES1 enhanced SRC phosphorylation. This activated SRC kinase was found to be responsible
  for the enhanced phosphorylation of STAT3. The HES1 and HES5 proteins associate
  with and facilitate the complex formation between JAK2 and STAT3, thus promoting
  STAT3 phosphorylation and activation. The activated STAT3 translocates from the
  cytoplasm to the nucleus and induces transcriptional activation of target gene expression
  (including HIF1A).  Please access this pathway at [NetSlim](http://www.netpath.org/netslim/notch_pathway.html)
  database.  If you use this pathway, please cite following paper: Kandasamy, K.,
  Mohan, S. S., Raju, R., Keerthikumar, S., Kumar, G. S. S., Venugopal, A. K., Telikicherla,
  D., Navarro, J. D., Mathivanan, S., Pecquet, C., Gollapudi, S. K., Tattikota, S.
  G., Mohan, S., Padhukasahasram, H., Subbannayya, Y., Goel, R., Jacob, H. K. C.,
  Zhong, J., Sekhar, R., Nanjappa, V., Balakrishnan, L., Subbaiah, R., Ramachandra,
  Y. L., Rahiman, B. A., Prasad, T. S. K., Lin, J., Houtman, J. C. D., Desiderio,
  S., Renauld, J., Constantinescu, S. N., Ohara, O., Hirano, T., Kubo, M., Singh,
  S., Khatri, P., Draghici, S., Bader, G. D., Sander, C., Leonard, W. J. and Pandey,
  A. (2010). NetPath: A public resource of curated signal transduction pathways. <i>Genome
  Biology</i>. 11:R3.'
last-edited: 2025-03-04
ndex: ae9b9d57-8b5f-11eb-9e72-0ac135e8bacf
organisms:
- Homo sapiens
redirect_from:
- /index.php/Pathway:WP61
- /instance/WP61
- /instance/WP61_r137144
revision: r137144
schema-jsonld:
- '@context': https://schema.org/
  '@id': https://wikipathways.github.io/pathways/WP61.html
  '@type': Dataset
  creator:
    '@type': Organization
    name: WikiPathways
  description: 'The Notch pathway is an evolutionally conserved signaling pathway
    which plays an important role in diverse developmental and physiological processes.
    These include cell-fate determination, tissue patterning and morphogenesis, cell
    differentiation, proliferation and cell death. The Notch pathway is named after
    the Drosophila mutants that showed irregular notches of missing tissue at the
    insect wing blade tips. The Notch gene was cloned in 1985. Proteins of the Notch
    families are single-pass transmembrane proteins that function both as cell surface
    receptors and nuclear transcriptional regulators. Four Notch receptors (Notch
    1-4) have been identified in mammals. Mature Notch receptors are non-covalent
    heterodimers consisting of an extracellular subunit (NEC) and a transmembrane
    subunit (NTM). NEC possess multiple EGF-like repeats and three specialized Lin-Notch
    repeats (LNR) that forms a tight hydrophobic interaction with extracellular stump
    of NTM. This region masks an A disintegrin and metalloprotease (ADAM) cleavage
    site. The region where these two subunits interact is called the heterodimerization
    domain (HD). Notch ligands are also transmembrane proteins with multiple EGF-like
    repeats, a short cytoplasmic tail and a specialized delta-serrate-lag2 (DSL) domain
    at the N-terminus. There are five canonical Notch ligands i.e. Jagged (JAG1 and
    JAG2), Delta-like (DLL1, DLL3, DLL4) in mammals. Notch signaling activation occurs
    upon ligand-receptor binding, which are expressed on two adjacent cells. Ligand
    binding causes dissociation of NEC from NTM, unmasking the ADAM cleavage site.
    The NEC fragment is trans- endocytosed into the ligand expressing cells. The full-length
    receptor minus the NEC fragment is cleaved at the membrane by ADAM17 generating
    an intermediate, Notch extracellular truncation (NEXT). This is further cleaved
    by Î³-secretase that generates an active Notch intracellular fragment (NIC) or
    Notch intracellular domain (NICD). The Î³-secretase complex is composed of PSEN1,
    PSEN2, PSENEN, NCSTN and APH1 (A or B). Following these two cleavage steps, the
    NICD is released into the cytoplasm and translocates into the nucleus to regulate
    transcription of Notch target genes. Upon translocation into the nucleus, NICD
    binds to RBPJ which is a constitutive repressor of Notch signaling. RBPJ represses
    Notch target gene expression by recruiting a co-repressor complex, which includes
    NCOR1, NCOR2, SNW1, CIR, HDAC1, HDAC2, SPEN and FHL1 and SAP30. NICD binding to
    RBPJ replaces the co-repressor complex with a co-activator complex which includes
    MAML1-3, EP300 and SNW1. Primary Notch target genes include two families of transcriptional
    factors Hes, including HES1 and HES5 as well as Hey including HEY1 and HEY2. Other
    Notch target genes include CCND1, CDKN1A, GATA3 and PTCRA. CNTN1 acts as a functional
    ligand of Notch. This trans-extracellular interaction causes Î³-secretase-dependent
    nuclear translocation of the NICD. This signaling is involved in oligodendrocyte
    precursor cell differentiation and upregulation of myelin-related protein MAG.
    In addition to the canonical Notch pathway, there is increasing evidence showing
    RBPJ independent non-canonical pathways. been fully characterized. Physical interaction
    of NOTCH-1IC with LCK- PI3K may mediate non-nuclear cross-talk with AKT, leading
    to survival signaling. Notch stimulation through AKT pathway leads to down regulation
    of MYC expression. Activation of SRC/STAT3 pathway by Notch signaling is dependent
    on the expression of Notch effector HES1 transcription factor. The induction of
    HES1 enhanced SRC phosphorylation. This activated SRC kinase was found to be responsible
    for the enhanced phosphorylation of STAT3. The HES1 and HES5 proteins associate
    with and facilitate the complex formation between JAK2 and STAT3, thus promoting
    STAT3 phosphorylation and activation. The activated STAT3 translocates from the
    cytoplasm to the nucleus and induces transcriptional activation of target gene
    expression (including HIF1A).  Please access this pathway at [NetSlim](http://www.netpath.org/netslim/notch_pathway.html)
    database.  If you use this pathway, please cite following paper: Kandasamy, K.,
    Mohan, S. S., Raju, R., Keerthikumar, S., Kumar, G. S. S., Venugopal, A. K., Telikicherla,
    D., Navarro, J. D., Mathivanan, S., Pecquet, C., Gollapudi, S. K., Tattikota,
    S. G., Mohan, S., Padhukasahasram, H., Subbannayya, Y., Goel, R., Jacob, H. K.
    C., Zhong, J., Sekhar, R., Nanjappa, V., Balakrishnan, L., Subbaiah, R., Ramachandra,
    Y. L., Rahiman, B. A., Prasad, T. S. K., Lin, J., Houtman, J. C. D., Desiderio,
    S., Renauld, J., Constantinescu, S. N., Ohara, O., Hirano, T., Kubo, M., Singh,
    S., Khatri, P., Draghici, S., Bader, G. D., Sander, C., Leonard, W. J. and Pandey,
    A. (2010). NetPath: A public resource of curated signal transduction pathways.
    <i>Genome Biology</i>. 11:R3.'
  keywords:
  - ADAM17
  - AKT1
  - APH1A
  - APH1B
  - CIR
  - CUL1
  - DLL1
  - DLL3
  - DLL4
  - DTX1
  - EP300
  - FBXW7
  - FHL1
  - GSK3B
  - HDAC1
  - HDAC2
  - HES1
  - HES5
  - HES6
  - HEY1
  - HEY2
  - ITCH
  - JAG1
  - JAG2
  - JAK2
  - LCK
  - MAGEA1
  - MAML1
  - MAML2
  - MAML3
  - MTOR
  - NCOR1
  - NCOR2
  - NCSTN
  - NFKB1
  - NOTCH1
  - NOTCH2
  - NOTCH3
  - NOTCH4
  - NUMB
  - NUMBL
  - PIK3R1
  - PIK3R2
  - PSEN1
  - PSEN2
  - PSENEN
  - RBPJ
  - RING1
  - SAP30
  - SKP1
  - SNW1
  - SPEN
  - SRC
  - STAT3
  - TLE1
  license: CC0
  name: Notch signaling
seo: CreativeWork
title: Notch signaling
wpid: WP61
---