---
title: Tools
redirect_from: /index.php/Help:Tools_using_WikiPathways
---
<h2>Tools using WikiPathways</h2>
<p>These are some of the resources and software tools out there that make use of WikiPathways 
    content or provide novel ways to query and display our collection of 
    community-contributed pathways.</p>

<table cellspacing="5" cellpadding="5" align="left">
    <tr>
    <th><a href="https://www.agilent.com/" target="_blank" class="external " title="https://www.agilent.com/" rel="nofollow">Agilent Technologies</a>
    </th><td>Agilent Technologies produces commercial sequencing technology and analysis suites (GeneSpring and EnViz) that uses WikiPathways content via web services.
    </td></tr>
    <tr>
    <th><a href="http://bioconductor.org/packages/release/data/annotation/html/AHWikipathwaysDbs.html" target="_blank" class="external " title="http://bioconductor.org/packages/release/data/annotation/html/AHWikipathwaysDbs.html" rel="nofollow">AHWikipathwaysDbs</a>
    </th><td>The package provides a comprehensive mapping table of metabolites linked to Wikipathways pathways. The tables include HMDB, KEGG, ChEBI, Drugbank, PubChem compound, ChemSpider, KNApSAcK, and Wikidata IDs plus CAS and InChIKey. The tables are provided for each of the 25 species.
    </td></tr>
    <tr>
    <th><a href="https://www.altanalyze.org/" target="_blank" class="external " title="https://www.altanalyze.org/" rel="nofollow">AltAnalyze</a>
    </th><td>AltAnalyze is an easy-to-use application for the end-to-end analysis of microarry, single-cell (ICGS) and bulk RNA-Seq data. AltAnalyze uses WikiPathways as its official pathway source.
    </td></tr>
    <tr>
    <th><a href="https://bioportal.bioontology.org/" target="_blank" class="external " title="https://bioportal.bioontology.org/" rel="nofollow">BioPortal</a>
    </th><td>Use BioPortal to access and share ontologies that are actively used in biomedical communities. WikiPathways uses BioPortal to power its ontology tagging system and BioPortal lists all ontology-tagged pathways as <a href="http://bioportal.bioontology.org/resources" target="_blank" class="external " title="http://bioportal.bioontology.org/resources" rel="nofollow">resources</a>.
    </td></tr>
    <tr>
    <th><a href="https://pythonhosted.org/bioservices/" target="_blank" class="external " title="https://pythonhosted.org/bioservices/" rel="nofollow">BioServices</a>
    </th><td>BioServices is a Python package that provides access to many Bioinformatics Web Services (including WikiPathways) and a framework to easily implement Web Service wrappers (based on WSDL/SOAP or REST protocols).
    </td></tr>
    <tr>
    <th><a href="https://bioconductor.org/packages/clusterProfiler/" target="_blank" class="external " title="https://bioconductor.org/packages/clusterProfiler/" rel="nofollow">clusterProfiler</a>
    </th><td>The clusterProfiler R package supports GO and pathway analysis performed as overrepresentation or GSEA. The package includes identifier mapping functionality and handy plot functions customized for these analyses. The <a href="https://yulab-smu.top/biomedical-knowledge-mining-book/wikipathways-analysis.html" target="_blank" class="external " title="https://yulab-smu.top/biomedical-knowledge-mining-book/wikipathways-analysis.html" rel="nofollow">main vignette</a> describes how to use the latest WikiPathways GMT file in a pathway analysis.
    </td></tr>
    <tr>
    <th><a href="https://compath.scai.fraunhofer.de/" target="_blank" class="external " title="https://compath.scai.fraunhofer.de/" rel="nofollow">ComPath</a>
    </th><td>An integrative and extensible web application for exploring, analyzing, and curating pathway database. Published in <a href="https://doi.org/10.1038/s41540-018-0078-8" target="_blank" class="external " title="https://doi.org/10.1038/s41540-018-0078-8" rel="nofollow">[1]</a> npj Systems Biology and Applications.
    </td></tr>
    <tr>
    <th><a href="http://cpdb.molgen.mpg.de/" target="_blank" class="external " title="http://cpdb.molgen.mpg.de/" rel="nofollow">ConsensusPathDB</a>
    </th><td>ConsensusPathDB-human integrates functional interaction networks including complex protein-protein, metabolic, signaling and gene regulatory interaction networks in Homo sapiens. Data originate from currently 20 public resources for functional interactions, as well as interactions that we have curated from literature, including WikiPathways.
    </td></tr>
    <tr>
    <th><a href="https://proteomics.cancer.gov/assay-portal" target="_blank" class="external " title="https://proteomics.cancer.gov/assay-portal" rel="nofollow">CPTAC Assay Portal</a>
    </th><td>The goal of the portal is to widely disseminate highly characterized proteomic assays to the global research community, with access to SOPs, reagents, and assay characterization/validation data. You can browse assays by protein via clickable pathway models from WikiPathways. Learn how they incorpated <a href="/img_auth.php/f/f3/CPTAC_WikiPathways_Assays_Coordination.pdf" class="internal" title="CPTAC WikiPathways Assays Coordination.pdf">WikiPathways images with custom links on their site</a>.
    </td></tr>
    <tr>
    <th> <a href="https://cytoscape.org" target="_blank" class="external " title="https://cytoscape.org" rel="nofollow">Cytoscape</a>
    </th><td>Cytoscape is an open source software platform for visualizing complex-networks and integrating these with any type of attribute data. Apps are available for various kinds of problem domains, including bioinformatics, social network analysis, and semantic web. WikiPathways can be easily imported via web services using the <a href="http://apps.cytoscape.org/apps/wikipathways" target="_blank" class="external " title="http://apps.cytoscape.org/apps/wikipathways" rel="nofollow">WikiPathways app</a>. Pathways in gpml format created in PathVisio or WikiPathways can be loaded as either their native diagram format or as simplified network graphs, supporting either data visualization or network analysis and augmentation use cases. Tutorials are available at <a href="https://github.com/cytoscape/cytoscape-tutorials/wiki" target="_blank" class="external " title="https://github.com/cytoscape/cytoscape-tutorials/wiki" rel="nofollow">tutorials.cytoscape.org</a> and a detailed description of the app and its usage was published in the <a href="http://f1000research.com/articles/10.12688/f1000research.4254.2/doi" target="_blank" class="external " title="http://f1000research.com/articles/10.12688/f1000research.4254.2/doi" rel="nofollow">Cytoscape App Collection</a> in F1000Research.
    </td></tr>
    <tr>
    <th> <a href="https://decopath.scai.fraunhofer.de/" target="_blank" class="external " title="https://decopath.scai.fraunhofer.de/" rel="nofollow">DecoPath</a>
    </th><td>DecoPath is a web application to assist in the interpretation of results of pathway enrichment analysis.
    </td></tr>
    <tr>
    <th><a href="https://amp.pharm.mssm.edu/Enrichr/" target="_blank" class="external " title="https://amp.pharm.mssm.edu/Enrichr/" rel="nofollow">Enrichr</a>
    </th><td>Enrichr is a comprehensive gene set enrichment analysis web server. Includes WikiPathways as one of their data sources.
    </td></tr>
    <tr>
    <th><a href="http://www.esccatlas.org/" target="_blank" class="external " title="http://www.esccatlas.org/" rel="nofollow">ESCC Atlas</a>
    </th><td>ESCC ATLAS, a manually curated database that integrates genetic, epigenetic, transcriptomic, and proteomic Esophageal Squamous Cell Carcinoma-related genes from the published literature. ESCC ATLAS includes linkouts to WikiPathways.
    </td></tr>
    <tr>
    <th><a href="http://www.duccioknights.org/?page_id=169" target="_blank" class="external " title="http://www.duccioknights.org/?page_id=169" rel="nofollow">EuGene</a>
    </th><td>Eu.Gene Analyzer is an easy-to-use, stand-alone application that allows rapid and powerful microarray data analysis in the context of biological pathways. Its intuitive graphical user interface makes it an easy and flexible tool, even for the first-time user. Pathways can be downloaded from WikiPathways in EuGene's .pwf format.
    </td></tr>
    <tr>
    <th><a href="https://www.evinet.org/" target="_blank" class="external " title="https://www.evinet.org/" rel="nofollow">EviNet</a>
    </th><td>EviNet is a web resource that provides an easily run interface to network enrichment analysis for exploration of novel, experimentally defined gene sets.
    </td></tr>
    <tr>
    <th><a href="https://genetrail2.bioinf.uni-sb.de/" target="_blank" class="external " title="https://genetrail2.bioinf.uni-sb.de/" rel="nofollow">GeneTrail2</a>
    </th><td>GeneTrail2 is a web-interface providing access to different tools for the statistical analysis of molecular signatures. GeneTrail2 allows the integrated analysis of transcriptomic, miRNomic, genomic, and proteomic datasets. It offers multiple statistical tests, a large number of predefined reference sets, as well as a comprehensive collection of biological categories and enables direct comparisons between the computed results.
    </td></tr>
    <tr>
    <th><a href="https://en.wikipedia.org/wiki/Portal:Gene_Wiki" target="_blank" class="external " title="https://en.wikipedia.org/wiki/Portal:Gene_Wiki" rel="nofollow">Gene Wiki</a>
    </th><td>Gene Wiki is a subset of Wikipedia articles cover each and every human gene. These are curated and annoated by a collaborative group of biology domain experts. The content includes links between Gene Wiki and WikiPathways.
    </td></tr>
    <tr>
    <th><a href="http://www.genmapp.org/go_elite/go_elite.html" target="_blank" class="external " title="http://www.genmapp.org/go_elite/go_elite.html" rel="nofollow">GO-Elite</a>
    </th><td>GO-Elite is a software tool designed to identify a minimal non-redundant set of Gene Ontology (GO) biological terms or pathways to describe a particular set of genes. GO-Elite uses WikiPathways as its official pathway source. GO-Elite is also available as a Cytoscape plugin.
    </td></tr>
    <tr>
    <th><a href="https://biit.cs.ut.ee/gprofiler/gost" target="_blank" class="external " title="https://biit.cs.ut.ee/gprofiler/gost" rel="nofollow">g:Profiler</a>
    </th><td>g:Profiler is a public web server for characterising and manipulating gene lists.
    </td></tr>
    <tr>
    <th><a href="https://maayanlab.cloud/Harmonizome/dataset/Wikipathways+Pathways" target="_blank" class="external " title="https://maayanlab.cloud/Harmonizome/dataset/Wikipathways+Pathways" rel="nofollow">Harmonizome</a>
    </th><td>To facilitate access to and learning from biomedical data, Harmonizome was created as a collection of information about genes and proteins from 112 datasets provided by 65 online resources. It maintains a set of pages and bioinformatics resources related to the use of WikiPathways in enrichment analysis, including GMT files, Gene-Attribute Matrix, plots characterizing the overlap of gene sets, API access, example Python code, etc.
    </td></tr>
    <tr>
    <th><a href="https://homer.ucsd.edu/homer/" target="_blank" class="external " title="http://homer.ucsd.edu/homer/" rel="nofollow">HOMER</a>
    </th><td>HOMER (Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif Discovery and next-gen sequencing analysis. It is a collection of command line programs for unix-style operating systems written in Perl and C++. HOMER includes a wide range of "ontologies" for functional enrichment analysis, including WikiPathways.
    </td></tr>
    <tr>
    <th><a href="https://eweitz.github.io/ideogram/related-genes" target="_blank" class="external " title="https://eweitz.github.io/ideogram/related-genes" rel="nofollow">Ideogram related genes</a>
    </th><td>Tool for chromosome visualization that integrates interactions from WikiPathways. Used in <a href="https://singlecell.broadinstitute.org/single_cell" target="_blank" class="external " title="https://singlecell.broadinstitute.org/single_cell" rel="nofollow">Single Cell Portal</a>
    </td></tr>
    <tr>
    <th> <a href="https://github.com/gladstone-institutes/Interactive-Enrichment-Analysis" target="_blank" class="external " title="https://github.com/gladstone-institutes/Interactive-Enrichment-Analysis" rel="nofollow">Interactive Enrichment Analysis</a>
    </th><td>Interactive Enrichment Analysis provides user-friendly interactive tools to perform enrichment analysis for multiple datasets across multiple public databases. Databases are updated monthly, and the tool allows for analysis of multiple separate datasets at once. 
    </td></tr>  
    <tr>
    <th><a href="http://impala.molgen.mpg.de//" target="_blank" class="external " title="http://impala.molgen.mpg.de//" rel="nofollow">IMPaLA</a>
    </th><td>IMPaLA is a web tool, developed for integrated pathway analysis of metabolomics data alongside gene expression or protein abundance data. It works through extending over-representation and enrichment analyses to multiple data types. It uses pathway data from several different online pathway databases, including WikiPathways.
    </td></tr>
    <tr>
    <th><a href="http://irndb.org" target="_blank" class="external " title="http://irndb.org" rel="nofollow">IRNdb</a>
    </th><td>IRNdb is a database that combines microRNA, PIWI-interacting RNAs, and long non-coding RNA information with immunologically relevant target genes. The database is intended to advance research on the influence of ncRNAs on immunological processes, and includes WikiPathways as a data source.
    </td></tr>
    <tr>
    <th><a href="https://iquery.ndexbio.org/" target="_blank" class="external " title="http://iquery.ndexbio.org/" rel="nofollow">IQuery at NDEx</a>
    </th><td>NDEx is a network database for community-contrinbuted and reference networks. IQuery is a enrichment-based seach tool for networks and pathways in NDEx via gene lists. IQuery includes WikiPathways in its default tab of serach results, plus it includes a "Pathway Figures" tab for our <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02181-2" target="_blank" class="external " title="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02181-2" rel="nofollow">OCR-extraction project for published pathway figures</a>.
    </td></tr>
    <tr>
    <th><a href="http://lynx.ci.uchicago.edu/" target="_blank" class="external " title="http://lynx.ci.uchicago.edu/" rel="nofollow">Lynx: an Integrated Systems Biology Platform</a>
    </th><td>Lynx is a database and knowledge extraction engine for integrative medicine.
    </td></tr>
    <tr>
    <th><a href="https://www.ebi.ac.uk/metabolights/" target="_blank" class="external " title="http://www.ebi.ac.uk/metabolights/" rel="nofollow">MetaboLights</a>
    </th><td>MetaboLights is a database for Metabolomics experiments and derived information. It lists more then 20 thousand metabolites and for each in which pathways they are found, deriving from pathways from WikiPathways and Reactome.
    </td></tr>
    <tr>
    <th><a href="https://mpd.bioinf.uni-sb.de/" target="_blank" class="external " title="https://mpd.bioinf.uni-sb.de/" rel="nofollow">miRNA Pathway Dictionary Database (miRPathDB)</a>
    </th><td>miRPathDB aims to augment available target pathway web-servers by providing researches easy access to the information which pathways are regulated by a miRNA, which miRNAs target a pathway and how specific the regulations are.
    </td></tr>
    <tr>
    <th><a href="http://www.ncbi.nlm.nih.gov/Structure/biosystems/docs/biosystems_how_to.html" target="_blank" class="external " title="http://www.ncbi.nlm.nih.gov/Structure/biosystems/docs/biosystems_how_to.html" rel="nofollow">NCBI Biosystems</a>
    </th><td>The NCBI BioSystems Database is a centralized repository containing biological pathways from several resources and connects those with associated literature, molecular and chemical data throughout the Entrez system. This makes it possible to link to WikiPathways directly from NCBI pages. Whether you are browsing Gene, Pubchem or Structure entities at NCBI, you will find relevant summaries, images and links to WikiPathways. They are listed as "Pathways from BioSystems" under the "General gene information" section (<a href="http://www.ncbi.nlm.nih.gov/gene/4524#general-gene-info" target="_blank" class="external " title="http://www.ncbi.nlm.nih.gov/gene/4524#general-gene-info" rel="nofollow">example</a>). You can also view information about a single pathway on the <a href="http://www.ncbi.nlm.nih.gov/biosystems/198778?Sel=geneid:4524#show=genes" target="_blank" class="external " title="http://www.ncbi.nlm.nih.gov/biosystems/198778?Sel=geneid:4524#show=genes" rel="nofollow">summary pages</a> and browse our pathway content by <a href="http://www.ncbi.nlm.nih.gov/Structure/flink/flink.cgi?cmd=linkto&amp;fromdb=biosystems&amp;todb=gene&amp;linkname=biosystems_gene&amp;query_key=8" target="_blank" class="external " title="http://www.ncbi.nlm.nih.gov/Structure/flink/flink.cgi?cmd=linkto&amp;amp;fromdb=biosystems&amp;amp;todb=gene&amp;amp;linkname=biosystems_gene&amp;amp;query_key=8" rel="nofollow">gene</a>, <a href="http://www.ncbi.nlm.nih.gov/Structure/flink/flink.cgi?cmd=linkto&amp;fromdb=biosystems&amp;todb=pccompound&amp;linkname=biosystems_pccompound&amp;query_key=8" target="_blank" class="external " title="http://www.ncbi.nlm.nih.gov/Structure/flink/flink.cgi?cmd=linkto&amp;amp;fromdb=biosystems&amp;amp;todb=pccompound&amp;amp;linkname=biosystems_pccompound&amp;amp;query_key=8" rel="nofollow">pubchem</a> or <a href="http://www.ncbi.nlm.nih.gov/Structure/flink/flink.cgi?cmd=linkto&amp;fromdb=biosystems&amp;todb=structure&amp;linkname=biosystems_structure&amp;query_key=8" target="_blank" class="external " title="http://www.ncbi.nlm.nih.gov/Structure/flink/flink.cgi?cmd=linkto&amp;amp;fromdb=biosystems&amp;amp;todb=structure&amp;amp;linkname=biosystems_structure&amp;amp;query_key=8" rel="nofollow">structure</a> frequency.
    </td></tr>
    <tr>
    <th><a href="http://www.neuinfo.org/" target="_blank" class="external " title="http://www.neuinfo.org/" rel="nofollow">NIF</a>
    </th><td>The NIF (Neuroscience Information Framework) is a dynamic inventory of web-based neuroscience resources. WikiPathways is included as one of the resources.
    </td></tr>
    <tr>
    <th><a href="http://omnipathdb.org" target="_blank" class="external " title="http://omnipathdb.org" rel="nofollow">OmniPath</a>
    </th><td>OmniPath is a network of signaling pathways intending to combine all high quality, manually curated efforts.
    </td></tr>
    <tr>
    <th><a href="https://pypi.org/project/pathme/" target="_blank" class="external " title="https://pypi.org/project/pathme/" rel="nofollow">PathMe</a>
    </th><td>The primary goal of PathMe is to convert KEGG, Reactome, and WikiPathways to Biological Expression Language (BEL) as a pivotal integration schema to evaluate pathway cross-talks and consensus across multiple databases.
    </td></tr>
    <tr>
    <th><a href="http://compbio.sph.harvard.edu/hidelab/pathprint/Pathprint.html" target="_blank" class="external " title="http://compbio.sph.harvard.edu/hidelab/pathprint/Pathprint.html" rel="nofollow">Pathprint</a>
    </th><td>Pathprinting integrates pathway curation, profiling methods, and public repositories, to represent any expression profile as a ternary score (-1, 0, +1) in a standardized pathway panel.
    </td></tr>
    <tr>
    <th> <a href="https://pathvisio.org" target="_blank" class="external " title="http://pathvisio.org" rel="nofollow">PathVisio</a>
    </th><td> PathVisio is a free open-source pathway analysis and drawing software which allows drawing, editing, and analyzing biological pathways. It is developed in Java and can be extended with plugins. PathVisio is the pathway editor for WikiPathways, a community-curated pathway database enabling collaborative pathway editing.
    </td></tr>
    <tr>
    <th><a href="https://www.pathwaycommons.org/" target="_blank" class="external " title="http://www.pathwaycommons.org/" rel="nofollow">Pathway Commons</a>
    </th><td>Pathway Commons aims to store and disseminate knowledge about biological pathways, and includes pathway data from multiple sources, including WikiPathways.
    </td></tr>
    <tr>
    <th><a href="https://wikipathways.org/pathway-finder/?database=mirtarbase&amp;identifiers=hsa-miR-148a-3p,hsa-miR-21-3p,hsa-miR-29a-3p,hsa-miR-20a-5p" target="_blank" class="external " title="http://wikipathways.org/pathway-finder/?database=mirtarbase&amp;amp;identifiers=hsa-miR-148a-3p,hsa-miR-21-3p,hsa-miR-29a-3p,hsa-miR-20a-5p" rel="nofollow">Pathway Finder</a>
    </th><td>Query pathways based on miRNA targeting.
    </td></tr>
    <tr>
    <th><a href="http://pcxn.org" target="_blank" class="external " title="http://pcxn.org" rel="nofollow">PCxN</a>
    </th><td>Generates a co-activity map across pathways, assessing the correlation based on gene content within and across pathway resources.
    </td></tr>
    <tr>
    <th><a href="https://www.proteomicsdb.org/analytics/ptmNavigator" target="_blank" class="external " title="https://www.proteomicsdb.org/analytics/ptmNavigator" rel="nofollow">PTMNavigator</a>
    </th><td>PTMNavigator is a tool for interactive visualization of differentially regulated post-translational modifications in cellular signaling pathways, including WikiPathways. It is part of <a href="https://www.proteomicsdb.org/" target="_blank" class="external" title="https://www.proteomicsdb.org/" rel="nofollow">ProteomicsDB</a>. 
    </td></tr>
    <tr>
    <th><a href="https://github.com/kozo2/pywikipathways" target="_blank" class="external " title="https://github.com/kozo2/pywikipathways" rel="nofollow">pywikipathways</a>
    </th><td>Python client package for the WikiPathways webservice API.
    </td></tr>
    <tr>
    <th><a href="https://github.com/bonohu/qpx" target="_blank" class="external " title="https://github.com/bonohu/qpx" rel="nofollow">QPX</a>
    </th><td>Quest for Pathways with eXpression (QPX) is a tool that displays WikiPathways pathways and attribute tables for selected nodes in Jupyter notebook.
    </td></tr>
    <tr>
    <th><a href="https://github.com/Mathelab/RaMP-DB/" target="_blank" class="external " title="https://github.com/Mathelab/RaMP-DB/" rel="nofollow">RaMP</a>
    </th><td>Combines biological pathways from KEGG, Reactome, WikiPathways, and HMDB. For consistent and comprehensive analysis, RaMP enables batch and complex queries (e.g. list all metabolites involved in glycolysis and lung cancer), can readily be integrated into pathway analysis tools, and supports pathway overrepresentation analysis given a list of genes and/or metabolites of interest.
    </td></tr>
    <tr>
    <th><a href="https://cran.r-project.org/web/packages/rSEA/vignettes/rSEA_vignette.html" target="_blank" class="external " title="https://cran.r-project.org/web/packages/rSEA/vignettes/rSEA_vignette.html" rel="nofollow">rSEA</a>
    </th><td>rSEA is a novel paradigm for simultaneous enrichment analysis of feature-sets. It combines the pre-existing self-contained and competitive approaches by defining a unified null hypothesis that includes the null hypotheses of both approaches as special cases.
    </td></tr>
    <tr>
    <th><a href="https://bioconductor.org/packages/release/bioc/html/rWikiPathways.html" target="_blank" class="external " title="https://bioconductor.org/packages/release/bioc/html/rWikiPathways.html" rel="nofollow">rWikiPathways</a>
    </th><td>R package for the WikiPathways webservice API.
    </td></tr>
    <tr>
    <th><a href="http://101.200.211.232/skeletongenetics/" target="_blank" class="external " title="http://101.200.211.232/skeletongenetics/" rel="nofollow">SkeletonGenetics</a>
    </th><td>Database for genes and mutations related to genetic skeletal disorders. Displays WikiPathways data via WebGestalt.
    </td></tr>
    <tr>
    <th><a href="http://snplogic.org/" target="_blank" class="external " title="http://snplogic.org/" rel="nofollow">SNPLogic</a>
    </th><td>The purpose of SNPLogic is to provide comprehensive interactive SNP annotation, selection and prioritization system for focused genotyping projects and/or analysis and interpretation of SNP data. The user can create one or more SNP lists and populate them by adding SNPs either directly or by picking genes, pathways, chromosomal locations or by uploading own SNP lists.
    </td></tr>
    <tr>
    <th><a href="https://www.thermofisher.com/nl/en/home/life-science/microarray-analysis/microarray-analysis-instruments-software-services/microarray-analysis-software/affymetrix-transcriptome-analysis-console-software.html" target="_blank" class="external " title="https://www.thermofisher.com/nl/en/home/life-science/microarray-analysis/microarray-analysis-instruments-software-services/microarray-analysis-software/affymetrix-transcriptome-analysis-console-software.html" rel="nofollow">Transcriptome Analysis Console (TAC) Software</a>
    </th><td>The TAC Software enables you to go beyond simple identification of differential expression by providing powerful, interactive visualizations. TAC allows you to visualize data on the pathway diagrams and calculate pathway metrics.
    </td></tr>
    <tr>
    <th><a href="http://bidd.nus.edu.sg/BIDD-Databases/TTD/TTD_Pathway.asp" target="_blank" class="external " title="http://bidd.nus.edu.sg/BIDD-Databases/TTD/TTD_Pathway.asp" rel="nofollow">Therapeutic Targets Database</a>
    </th><td>TTD is a database to provide information about the known and explored therapeutic protein and nucleic acid targets, the targeted disease, pathway information and the corresponding drugs directed at each of these targets.
    </td></tr>
    <tr>
    <th><a href="https://www.webgestalt.org" target="_blank" class="external " title="http://www.webgestalt.org" rel="nofollow">WebGestalt</a>
    </th><td>WebGestalt is a "WEB-based GEne SeT AnaLysis Toolkit". It is designed for functional genomic, proteomic and large-scale genetic studies from which large number of gene lists (e.g. differentially expressed gene sets, co-expressed gene sets etc) are continuously generated. WebGestalt incorporates information from different public resources, including WikiPathways, and provides an easy way for biologists to make sense out of gene lists.
    </td></tr>
    <tr>
    <th><a href="https://wikidata.org" target="_blank" class="external " title="http://wikidata.org" rel="nofollow">Wikidata</a>
    </th><td>Wikidata is a free and open knowledge base that can be read and edited by both humans and machines.  We have added interactive pathway pages for WikiPathways content via Scholia (<a href="https://tools.wmflabs.org/scholia/pathway/" target="_blank" class="external " title="https://tools.wmflabs.org/scholia/pathway/" rel="nofollow">examples</a>).
    </td></tr>
    <tr>
<th><a href="https://github.com/Thakar-Lab/wikinetworks" target="_blank" class="external " title="https://github.com/Thakar-Lab/wikinetworks" rel="nofollow">WikiNetworks</a>
</th><td>WikiNetworks is a user-friendly Python package to process pathways from the WikiPathways database into machine-readable network representations. This enables the use of high-quality WikiPathways resources for network-based topological analysis of high-throughput data. WikiNetworks allows users to (a) download manually created gpml files from WikiPathways by specifying pathway id and (b) convert them to machine-interpretable networks suitable for use with pathway/network analysis tools and (c) for import into pathway visualization tools.
</td></tr>
    <tr>
    <th><a href="https://wikipedia.org" target="_blank" class="external " title="http://wikipedia.org" rel="nofollow">Wikipedia</a>
    </th><td>The free encyclopedia that anyone can edit. In collaboration with MCB and GeneWiki, we have added interactive pathway maps using WikiPathways (<a href="http://en.wikipedia.org/wiki/DPYD" target="_blank" class="external " title="http://en.wikipedia.org/wiki/DPYD" rel="nofollow">example</a>).
    </td></tr>
    <tr>
    <th><a href="https://github.com/YuLab-SMU/wikiprofiler" target="_blank" class="external " title="https://github.com/YuLab-SMU/wikiprofiler" rel="nofollow">wikiprofiler</a>
    </th><td>Queries online 'WikiPathway' graphics and allows mapping user data (e.g., expression values) on the graph.
        The package designs a grammar of graphic syntax that using pipe operator to add graphic layer.
    </td></tr>
</table>

<p>
<b>If you know of a tool or resource that makes use of our curated pathway content, let us know via our <a href="https://github.com/wikipathways/wikipathways-help/discussions" target="_blank" class="external " title="https://github.com/wikipathways/wikipathways-help/discussions" rel="nofollow">helpdesk</a> so we can add them to the list.</b>
</p>
